README.rst

```diff
@@ -56,3 +56,9 @@ Infrastructure
 --------------
 
 The site is deployed on `Netlify <https://www.netlify.com>`_ from the ``master`` branch of this repository, fully regenerated on every build. The domain is registered by `@nkantar <https://github.com/nkantar>`_ through `Hover <https://www.hover.com>`_, and SSL is provided by `Let's Encrypt <https://letsencrypt.org>`_.
+
+
+Code of Conduct
+---------------
+
+Everyone contributing to this repository is bound by the Code of Conduct employed by the conference. The full text can be found in the ``CODE_OF_CONDUCT.md`` file.
```

Comment:
I'd like to add the following blurb (more or less) at the beginning, middle, or end of this section:

> PyBeach employs the [Python Software Foundation's Code of Conduct](https://www.python.org/psf/codeofconduct/).